src/collapse/perf.rs

```diff
@@ -261,16 +260,8 @@ impl PerfState {
 
             // --inline flag
             if self.opt.show_inline && can_un_inline(module) {
-                match self.un_inline(pc, module) {
-                    Ok(()) => {
-                        return;
-                    }
-                    Err(e) => {
-                        eprintln!(
-                            "Error attempting to un-inline {} in {}, caused by: {}",
-                            pc, module, e
-                        );
-                    }
+                if self.un_inline(pc, module) {
+                    return;
```

Comment:
We should check what the Perl version does here -- whether it always returns when you have `--inline` specified (i.e., doesn't fall back to another type of resolution) or not.

Reply:
Yeah, the Perl version never falls back to another type of resolution, so we probably shouldn't either. The Perl version will still be different when addr2line returns an error message since the error will be included on the stack, where we log the error and nothing gets added to the stack.